src/material/checkbox/_checkbox-common.scss

```diff
@@ -0,0 +1,515 @@
+@use 'sass:math';
+@use '@angular/cdk';
+@use '../core/tokens/m2/mdc/checkbox' as tokens-mdc-checkbox;
+@use '../core/tokens/token-utils';
+
+$_path-length: 29.7833385;
+$_transition-duration: 90ms;
+$_icon-size: 18px;
+$_mark-stroke-size: math.div(2, 15) * $_icon-size;
+$_indeterminate-checked-curve: cubic-bezier(0.14, 0, 0, 1);
+$_indeterminate-change-duration: 500ms;
+$_enter-curve: cubic-bezier(0, 0, 0.2, 1);
+$_exit-curve: cubic-bezier(0.4, 0, 0.6, 1);
+
+// Structural styles for a checkbox. To be shared with the selection list.
```

Comment:
Can you file a issue for sharing structural styles with selection list and refer to it here

Reply:
Ah, I ended up switching the list over to these styles in this PR. Will change the comment.